README.md

```diff
@@ -9,162 +9,22 @@ urlFragment: ms-identity-python-webapp
 ---
 # Integrating Microsoft Identity Platform with a Python web application
 
-## About this sample
+This is a Python web application that uses the Flask framework and the Microsoft identity platform to sign in users and make authenticated calls to the Microsoft Graph API.
 
-> This sample is also available as a quickstart for the Microsoft identity platform:
-[Quickstart: Add sign-in with Microsoft to a Python web app](https://docs.microsoft.com/azure/active-directory/develop/web-app-quickstart?pivots=devlang-python)
+To get started with this sample, follow this tutorial:
+[Quickstart: Add sign-in with Microsoft to a Python web app](https://docs.microsoft.com/azure/active-directory/develop/web-app-quickstart?pivots=devlang-python).

```

Comment:
I love how [your new README](https://github.com/pamelafox/ms-identity-python-webapp/tree/style-fixes#readme) has been greatly simplified. Way to go, Pamela!

One suggestion here. The way your README looks, might have a careless reader to jump to the "Using PowerShell scripts" header. Do you consider presenting the two options equally? For example, by using bullet points?

```suggestion
To get started with this sample, you have two options.

* Using the Azure portal to create the Azure AD applications and related objects. Follow this tutorial:
  [Quickstart: Add sign-in with Microsoft to a Python web app](https://docs.microsoft.com/azure/active-directory/develop/web-app-quickstart?pivots=devlang-python).
* Using PowerShell scripts that automatically create the Azure AD applications and related objects (passwords, permissions, dependencies) for you, and then modify the applications' configuration files. To use this automation, follow the steps in the [App Creation Scripts README](./AppCreationScripts/AppCreationScripts.md).
```